testing/run_tests.py

```diff
@@ -279,16 +279,16 @@ def EnsureJavaTestsAreBuilt(android_out_dir):
   RunCmd(gn_command, cwd=buildroot_dir)
   RunCmd(ninja_command, cwd=buildroot_dir)
 
-def AssertExpectedJavaVersion():
+def AssertExpectedJavaVersion(java_path):
   EXPECTED_VERSION = '1.8'
   # `java -version` is output to stderr. https://bugs.java.com/bugdatabase/view_bug.do?bug_id=4380614
-  version_output = subprocess.check_output(['java', '-version'], stderr=subprocess.STDOUT)
+  version_output = subprocess.check_output([java_path, '-version'], stderr=subprocess.STDOUT)
```

Comment:
if you really want this, you'll probably need an env map too

Reply:
for JAVA_HOME

Reply:
Ahh yeah, with JAVA_HOME set everything is working. Thanks!